lib/mix/tasks/tidewave.install.ex

```diff
@@ -0,0 +1,161 @@
+defmodule Mix.Tasks.Tidewave.Install.Docs do
+  @moduledoc false
+
+  def short_doc do
+    "Installs `tidewave` into your project"
+  end
+
+  def example do
+    "mix igniter.install tidewave"
+  end
+
+  def long_doc do
+    """
+    #{short_doc()}
+
+    ## Example
+
+    ```bash
+    #{example()}
+    ```
+    """
+  end
+end
+
+if Code.ensure_loaded?(Igniter) do
+  defmodule Mix.Tasks.Tidewave.Install do
+    @shortdoc "#{__MODULE__.Docs.short_doc()}"
+
+    @moduledoc __MODULE__.Docs.long_doc()
+    @plug_example """
+        if code_reloading? do
+          socket "/phoenix/live_reload/socket", Phoenix.LiveReloader.Socket
+        + if Code.ensure_loaded?(Tidewave) do
+        +   plug Tidewave
+        + end
```

Comment:
We are asking to place it before the `if code_reloading? do` block. Can you please update?